app/src/hooks/AuthHandler.ts

```diff
@@ -79,6 +79,8 @@ const AuthHandler: React.FC<{}> = () => {
         dispatch(globalActions.updateUsername({ username: userData.username }));
       }
 
+      dispatch(globalActions.updateUserBlockConfig(userData?.["block-config"]));
```

Comment:
Why explicitly add this? Isn't we already doing this?

Reply:
On first load, somehow AuthHandler was not getting block config and listeners changes were not applied